--- a/pypdfocr/pypdfocr_tesseract.py
+++ b/pypdfocr/pypdfocr_tesseract.py
@@ -95,8 +95,8 @@ def _is_version_uptodate(self):
                     ver_str = ver_str[:-3]

         # Iterate through the version dots
-        ver = [int(x) for x in ver_str.split('.')]
-        req = [int(x) for x in self.required.split('.')]
+        ver = [int(x) for x in ver_str.split('.') if x.isdigit()]
+        req = [int(x) for x in self.required.split('.') if x.isdigit()]

         # Aargh, in windows 3.02.02 is reported as version 3.02  
         # SFKM


